documentation/source/conf.py

```diff
@@ -32,74 +38,44 @@
 extensions = ["breathe", "sphinx.ext.mathjax", "sphinx.ext.ifconfig"]
 
 read_the_docs_build = os.environ.get("READTHEDOCS", None) == "True"
-travis_build = os.environ.get("TRAVIS_CI", None) == "True"
-
-# Get a description of the current position. Use Popen for 2.6 compat
-git_tag = subprocess.Popen(["git", "describe", "--tags"], stdout=subprocess.PIPE).communicate()[0]
 
-# convert from bytes to string
-git_tag = git_tag.decode("ascii")
-
-if travis_build:
-
-    # Don't attempt to set the path as breathe is installed to virtualenv on travis
-
-    # Set values with simple strings
-    version = "'travis'"
-    release = "'travis'"
-    documentation_build = "travis"
-
-elif read_the_docs_build:
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+# The short X.Y version.
+version = "'unknown'"
+# The full version, including alpha/beta/rc tags.
+release = "'unknown'"
 
-    # On RTD we'll be in the 'source' directory
-    sys.path.append("../../")
+# Get a description of the current position. Use Popen for 2.6 compat
+git_tag = get_version("breathe")
+
+# Check if it matches a pure tag number vX.Y.Z, rather than vX.Y.Z-91-g8676988 which is how
+# non-tagged commits are described (ie. relative to the last tag)
+if re.match(r"^\d+\.\d+\.\d+$", git_tag):
+    # git_tag is a pure version number (no subsequent commits)
+    version = "v" + git_tag
+    release = "v" + git_tag
```

Comment:
local testing revealed that the package metadata (used from this PR's changes) did not output a `v` prefix.

Reply:
Presumably `get_version` already strips that internally? Breathe has used the `v` prefix for git tags for a long, long time now.